### PR TITLE
Build Palindrome-Checker

### PR DESCRIPTION
**Pull request by royajamal**
"https://github.com/royajamal/Palindrome-Checker/assets/168343975/8e3b67f6-570a-47af-bf3a-45c0c4e8f27f"
Palindrome Checker  is our first project in JavaScript 

The Palindrome Checker is a web application that checks whether a given string is a palindrome. It removes non-alphanumeric characters, converts the string to lowercase, and compares it to its reverse to determine if it reads the same backward as forward.
In this i added 
HTML file
CSS file
JavaScript file
and linters

### CODE REVIEW

**Comment by shogof** [comment]
Hi @royajamal 👋,

Great job on the recent updates to the README file! Your project is shaping up nicely. After reviewing the changes, here are a few suggestions that could further enhance it:

1. **Project Screenshot Issue**: It seems like the project screenshot link is broken. Double-check the path to the image and ensure it's correctly referenced in the README.

2. **Live Link Addition**: Consider adding a live link to the project in the README. This allows visitors to quickly access and interact with your work.

3. **Installation Instructions**: Including clear installation instructions would greatly benefit users looking to set up the project locally. Outline the steps required to install dependencies and get everything up and running smoothly.

4. **Usage Section**: It would be helpful to include a usage section that explains how users can interact with your project. Providing examples of common commands or workflows can make it more accessible.

Additionally, **I noticed that the project doesn't pass all the freeCodeCamp tests**. Taking some time to review the requirements and ensure all test cases are met will strengthen the project's completeness.

Regarding your code comments, **making them clearer and more descriptive** would greatly improve readability. It can be challenging to understand what each comment refers to or describes. Being more specific will help anyone reading the code to follow your thought process easily.

On another note, your design choices are really coming together, and the professional look is impressive! 🌟

Keep up the great work, my dear. I'm excited to see how your project evolves with these updates!

Best regards,
Shegofa Developer

**Comment by royajamal** [comment]
@shogof 
Thanks for your feedback, I've edited my README.md file as you say and all of my code pass
FreeCodeCamp tests so now it's fixed please check it.

**Comment by shogof** [comment]
@royajamal
✅ **Status: Approved**
Wow, Roya jan! 🎉 You did an amazing job! You've successfully fulfilled all the requirements, and your project has been approved. 
**Optional:**
It would be fantastic if you could add a screenshot of your project to your readme file instead of just the link. Also, when sharing your live link, it just show your html structure and it don't show your great CSS design.
Great job! 👏

**Comment by royajamal** [comment]
@ahzamir
Hello, teacher I hope you are doing well. kindly review my palindrome project .
Thank you.

File: style.css
Comment (ahzamir) [inline style.css:1]: - Although you have added an impressive design to your project but your design is not responsive please add some more design to your project to make it responsive.
- Change your file name from `style.css` to `styles.css` as the same name you linked it in your HTML file

**Review by ahzamir — CHANGES_REQUESTED** [review]
### Hi @royajamal 🙋‍♀️,

> Excellent work! 💯 You have done a great job implementing all the project requirements. 🥳 Your feature is complete! Congratulations!!!🎊

- **[REQUIRED]** Share your project on this [GROUP](https://www.facebook.com/groups/kick-startdev), add a simple Proud Moment text and a screenshot of your project then besides that add your project link at the end of the text as a GitHub Source code reference and post.

- Share your Project on your LinkedIn account and add a proud moment text on it with a screenshot.

- Well Done great design Implemented. 🎯🎉👍
image



# Status: Required Changes 🔁
Make changes to your code and commit and request for a re-review.

AlmostThereReachGIF


 ## Highlights 🌞

🎉 Great Design Implemented 
✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
✔️ You have followed the Palindrome Checker project task and committed using the command line
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)
✔️Your code is clean
✔️Your repo has a professional README.md file
✔️Your repo have been reviewed by your partner
❌Your design is responsive

**Cheers 🥂 and Happy coding!!! 👯 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @ahzamir in your question so I can receive the notification.**

**Comment by royajamal** [comment]
@ahzamir
Thank you for your feedback . Now all of things is done please check it.

**Review by ahzamir — APPROVED** [review]
### Hi @royajamal 🙋‍♀️,

> Excellent work! 💯 You have done a great job implementing all the project requirements. 🥳 Your feature is complete! Congratulations!!!🎊

- **[REQUIRED]** Share your project on this [GROUP](https://www.facebook.com/groups/kick-startdev), add a simple Proud Moment text and a screenshot of your project then besides that add your project link at the end of the text as a GitHub Source code reference and post.

- Share your Project on your LinkedIn account and add a proud moment text on it with a screenshot.

- Well Done great design Implemented. 🎯🎉👍
image

# Status: Approved ✅
There is nothing to say other than to merge your branch. 🎉🎯✅👍😊🥳

WinkThumbsUpGIF (2)

 ## Highlights 🌞

🎉 Great Design Implemented 
✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
✔️ You have followed the Palindrome Checker project task and committed using the command line
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)
✔️Your repo has a professional README.md file
✔️Your code is clean
✔️Your design is impressive and responsive

**Cheers 🥂 and Happy coding!!! 👯 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @ahzamir in your question so I can receive the notification.**

**Pull request merged**